include/exadg/grid/mapping_deformation_structure.h

```diff
@@ -121,8 +121,12 @@ class DeformedMapping : public DeformedMappingBase<dim, Number>
         this->param.update_preconditioner &&
         time_step_number % this->param.update_preconditioner_every_time_steps == 0;
 
-      auto const iter = pde_operator->solve_nonlinear(
-        displacement, const_vector, 0.0 /* no mass term */, time, update_preconditioner);
+      auto const iter = pde_operator->solve_nonlinear(displacement,
+                                                      const_vector,
+                                                      0.0 /* no mass terms */,
+                                                      0.0 /* no mass terms */,
```

Comment:
why do you write two times "no mass terms"? I thought one of them refers to damping, so shouldn't one then write "no damping term" for one of the parameters?

Reply:
It is because both acceleration and damping terms lead to a scaled mass matrix. I could put `/* scaling_factor_mass_from_acceleration */` and /* scaling_factor_mass_from_velocity */ to make it more clear? For the current implementation and usage, `factor acceleration` and `factor damping` in my opinion would also work. Which one do you prefer?